Wash-Ups.md

```diff
@@ -0,0 +1,13 @@
+A wash-up can be held after any significant event that has happened where we believe actions can be taken so that we can better improve process moving forward.  This could be following an expedite issue or at the end of a large project.
+
+A new wash-up meeting template has been created that should help people to prepare for the meeting and ensure that all information on why the meeting is taking place has been filled in and circulated to all relevant parties in advance.
```

Comment:
change "that should help people to prepare" to "to prepare".

Maybe look at shortening this sentence too?